Yoast/wordpress-seo
# Add links table

## Summary

This PR can be summarized in the following changelog entry:
- Added database table to store links used in posts

## Test instructions

This PR can be tested by following these steps:

* Add some links to a post (internal and external). Make sure you have some internal links
* Save the post
* See the amount of links in the post overview being the same amount as used in the post. 

Fixes #7216
